docs/book/src/reference/versions.md

```diff
@@ -75,7 +75,7 @@ These diagrams show the relationships between components in a Cluster API releas
 | Kubernetes v1.23* |                      | ✓                 | ✓                 | ✓                 | ✓                 | ✓                 | ✓                 |
 | Kubernetes v1.24  |                      |                   |                   | ✓                 | ✓                 | ✓                 | ✓                 |
 | Kubernetes v1.25  |                      |                   |                   |                   | ✓                 | ✓                 | ✓                 |
-| Kubernetes v1.26  |                      |                   |                   |                   |                   |                   |                   |
+| Kubernetes v1.26  |                      |                   |                   |                   |                   | ✓                 | ✓                 |
```

Comment:
If I got it right from the test infra PR, we are adding test coverage for 1.26 also in 1.2, is that right? should we document this as well here?
(same below)

Reply:
Yes that make sense!